charts/values.yaml

```diff
@@ -0,0 +1,100 @@
+# Default values for growthbook.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: growthbook/growthbook
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: latest
+
+externalSecretName: growthbook-external-secret
+
+port:
+  frontendPort: 3000
+  backendPort: 3100
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+env:
+  - name: NODE_ENV
+    value: production
+  - name: MONGODB_URI
+    valueFrom:
+      secretKeyRef:
+        name: growthbook-external-secret
+        key: mongo-connection-uri
+  - name: JWT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: growthbook-external-secret
+        key: jwt-secret
+  - name: ENCRYPTION_KEY
+    valueFrom:
+      secretKeyRef:
+        name: growthbook-external-secret
+        key: encryption-key      
+  # Specify your app_origin, api_host
+  #- name: APP_ORIGIN
+  #  value: https://my-app-origin.io:443
+  #- name: API_HOST
+  #  value: https://my-api-host.io:443
+
+# Set your nodeSelector setting, following example uses AWS on_demand node.
+#nodeSelector:
+#  capacity-type: on_demand
+
+# Specify your storage class spec for persisting upload images to external file system
+#pvc:
+#  claimName: growthbook-efs-claim
+#  storageClassName: efs-sc
+#  storage: 5Gi
+
+volume:
+  name: uploads-persistent-storage
+  mountPath: /usr/local/src/app/packages/back-end/uploads
+
+service:
+  type: ClusterIP
+
+ingress:
+  name: growthbook-ingress
+  enabled: true
+  # Specify you app-origin, api-host without 'https://' prefix
+  #appOriginName: my-app-origin.io
+  #apiHostName: my-api-host.io
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+
+# Make sure you specify right mongodb credential in secret
+# Example below assumes prometheus is running at sys-monitoring namespace
+# For more configurations and explanations, visit https://github.com/bitnami/charts/tree/master/bitnami/mongodb
+# mongodb:
```

Comment:
We should be able to enable/disable mongo deploy dependency

Reply:
I added comment so that users can disable bitnami Mongodb chart.